### DISCUSSION TITLE
使用docker进行部署bfe，怎么设置mod_doh

### DISCUSSION THREAD

**Discussion by Ranwanglc** [bfenetworks/bfe#1106] [Q&A (问题和回答)]
我使用docker进行部署bfe的时候，启动后无法进行输入bfe命令，也无法进入容器进行修改bfe的配置文件，-d后台启动后可以查看8421接口页面在monitor 页面也无法发现mod_doh。docker hub的文档也十分有限，bfe的文档中也没有提到docker的部署情况下如何进行设置mod_doh。
另，我尝试使用go部署，但是当用启动命令的时候，也警告了maxprocs: Leaving GOMAXPROCS=1: CPU quota undefined

**Comment by youerning** [comment]
> 我使用docker进行部署bfe的时候，启动后无法进行输入bfe命令，也无法进入容器进行修改bfe的配置文件，-d后台启动后可以查看8421接口页面在monitor 页面也无法发现mod_doh。docker hub的文档也十分有限，bfe的文档中也没有提到docker的部署情况下如何进行设置mod_doh。
> 另，我尝试使用go部署，但是当用启动命令的时候，也警告了maxprocs: Leaving GOMAXPROCS=1: CPU quota undefined

默认的配置文件bfe.conf没有启用mod_doh，你可以找到
```
Modules = mod_access
``` 
这一行的下面加入
```
Modules = mod_doh
```
当加入这行之后，你可以在conf/mod_doh目录下找到mod_doh.conf进行配置
